vignettes/getting-started-with-teal.Rmd

```diff
@@ -35,75 +34,84 @@ library(teal)
 
 app <- init(
   data = teal_data(IRIS = iris, MTCARS = mtcars),
-  modules = example_module(),
-  header = "My first teal application"
+  modules = modules(
+    example_module("Module 1"),
+    example_module("Module 2")
+  ),
+  filter = teal_slices(
+    teal_slice(dataname = "IRIS", varname = "Species", selected = "setosa")
+  ),
+  title = build_app_title(title = "My first teal app"),
+  header = h3("My first teal application"),
+  footer = div(a("Powered by teal", href = "https://insightsengineering.github.io/teal/latest-tag/"))
 )
 
 if (interactive()) {
   shinyApp(app$ui, app$server)
 }
 ```
-<img src="images/example_app.png" alt="Example application" style="width: 100%;"/>
 
-As shown in the image above, this application consists of several distinct areas:
-
-* <span style="color: #FB6251;">Application Header</span> (title at the top): is the title of the application.
-* <span style="color: #3A88FE;">Teal Modules</span> (bar under the title): in this case a simple module named "example teal module".
-* <span style="color: #BF37F3;">Encoding Panel</span> (panel on the left hand side): Module specific UI components, in this case a drop-down to select a dataset name.
-* <span style="color: #96D35F;">Main Output Panel</span> (panel on the middle): The outputs of the module, for this example module the chosen dataset is displayed.
-* <span style="color: #1DC6FC;">Filter Panel</span> (panel on the right hand side): for filtering the data to be passed into all `teal` modules.
-
-### Encoding panel
```

Comment:
I thought it was okay because this will be the first landing page that many will start with. And `teal` has nothing to do with the `encoding` and `main` panel because that is how `teal.widgets` make our `teal.modules.*` packages built the module. And generally speaking, that content is meant for "module content" which might choose to not have "encoding and main output" format.

Reply:
Fair enough.

And I suppose we can also say the same thing about the Filter Panel section where can be covered more detail in filter-panel.Rmd and teal.slice getting started.

I'm fine with this.